Firestore/core/src/firebase/firestore/util/status.cc

```diff
@@ -1,28 +1,30 @@
-/* Copyright 2015 The TensorFlow Authors. All Rights Reserved.
+/*
+ * Copyright 2015, 2018 Google
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
 
-Licensed under the Apache License, Version 2.0 (the "License");
-you may not use this file except in compliance with the License.
-You may obtain a copy of the License at
+#include "Firestore/core/src/firebase/firestore/util/status.h"
 
-    http://www.apache.org/licenses/LICENSE-2.0
+namespace firebase {
+namespace firestore {
+namespace util {
 
-Unless required by applicable law or agreed to in writing, software
-distributed under the License is distributed on an "AS IS" BASIS,
-WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
-See the License for the specific language governing permissions and
-limitations under the License.
-==============================================================================*/
-
-#include "tensorflow/core/lib/core/status.h"
-#include <stdio.h>
```

Comment:
Isn't this include required for `snprintf`?

Reply:
I can't see any `snprintf` in here.

Also don't do that: use `[StringPrintf](https://github.com/firebase/firebase-ios-sdk/blob/master/Firestore/core/src/firebase/firestore/util/string_printf.h)` instead if you really must printf into a string.

Reply:
[Line 105](https://github.com/firebase/firebase-ios-sdk/pull/935/files#diff-f901ab23de521d6fff7877df5f8f22abR105).

Reply:
I've included stdio.h, and left it as snprintf. The function in question makes use of char*'s and char[]'s rather than std::string's. (We could convert it to strings at the cost of making more changes to original?)